### PR TITLE
WEBEU2 - Responsive Web Design I - Rui Hildt 🇧🇪 

### PR DESCRIPTION
**Pull request by ruihildt**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,16 +1,97 @@
-<!doctype html>
-
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
-
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (maximesalomon) [inline index.html]: 👍

File: css/index.css [hunk 2]
```diff
@@ -1,3 +1,413 @@
...
+    color: grey;
+}
+
+a:hover {
+    text-decoration: none;
+    color: rosybrown;
+}
+
+h1 {
+    font-size: 4.5em;
```
Comment (maximesalomon) [inline css/index.css]: Why em vs rem here?

Comment (maximesalomon) [reply css/index.css]: Capture d’écran 2019-04-29 à 18 42 58

Comment (ruihildt) [reply css/index.css]: That was value from my first code, didn't touch it but probably should have.

Regarding the alignment, I tried to be pixel perfect, but it didn't work out. I'll do some research on centering text in the future.

**Pull request merged**